chapters/hardware-software-interface/x86-architecture-family/drills/tasks/grumpy-jumps/README.md

```diff
@@ -0,0 +1,11 @@
+# Grumpy Jumps
+
+You will solve the exercises starting from the ```grumpy_jumps.asm``` file located in the ```drills/tasks/grumpy-jumps``` directory.
+
+1. Modify the values of the `EAX` and `EBX` registers so that when the program is run, the message `Well done!` is displayed. Follow the `TODO` comments.
+
+2. Why does the wrong message still appear? Modify the source so that the wrong message is not displayed anymore.
+
+> **TIP**: To determine the necessary values for the `EAX` and `EBX` registers, we recommend using GDB.
```

Comment:
```suggestion
You will solve the exercises starting from the ```grumpy_jumps.asm``` file located in the ```drills/tasks/grumpy-jumps``` directory.

1. Modify the values of the `eax` and `ebx` registers so that when the program is run, the message `Well done!` is displayed. Follow the `TODO` comments.

2. Why does the wrong message still appear? Modify the source so that the wrong message is not displayed anymore.

> **TIP**: To determine the necessary values for the `eax` and `ebx` registers, we recommend using GDB.
```